[irobot] Update documentation regarding button names (#17806)
openhab/openhab-addons: bundles/org.openhab.binding.irobot/README.md

## Discovery

Roombas on the same network will be discovered automatically, however in order to connect to them a password is needed.
The password is a machine-generated string, which is unfortunately not exposed by the original iRobot smartphone application, but it can be downloaded from the robot itself.
If no password is configured, the Thing enters "CONFIGURATION PENDING" state.
Now you need to perform authorization by pressing and holding the HOME/DOCK button on your robot until it plays series of tones (approximately 2 seconds).
The Wi-Fi indicator on the robot will flash for 30 seconds, the binding should automatically receive the password and go ONLINE.

After you've done this procedure you can write the password somewhere in case if you need to reconfigure your binding.

> It'snot known, however, whether the password is eternal or can change during factory reset.

Author: unchanged just one line per sentence.

If you have issues getting the password make sure there are no other devices like your smartphone communicating with the robot.
You can also try using [these python scripts](https://github.com/NickWaterton/Roomba980-Python) to get the password.

> *note* For file based configuration storing the password is essential.

Reviewer: I'm not sure how this is best formatted, but probably capitalize first letter and add a colon before the capital "F"?
```suggestion
*Note:* For file based configuration storing the password is essential.
```

Author: looked at some other bindings and by far the most use **NOTE**

Reviewer: 👍 I do think a colon is needed though as well, since the next word starts a new sentence.

Author: Done.

> **NOTE** For file based configuration storing the password is essential.

Reviewer: ```suggestion
**NOTE:** For file-based configuration, storing the password is essential.
```

> **NOTE:** For file based-configuration, storing the password is essential.

Reviewer: ```suggestion
**NOTE:** For file-based configuration, storing the password is essential.
```

> Once the password is populated in the Thing properties, you must copy that into the [config files](#irobotthings-example) in order for it to persist.

Reviewer: It won't be in Thing properties, but it can be found in the code tab, e.g.:
```yaml
UID: irobot:roomba:roomba960
label: Roomba
thingTypeUID: irobot:roomba
configuration:
  ipaddress: 192.168.1.63
  password: xxx
```

Author: Thanks, adjusted. Please check if this is clear enough, i struggle a little with the sentence.

> Once the password is populated in the MainUI Thing's codetab, you must copy that into the [config files](#irobotthings-example) in order for it to persist.

Reviewer: I don't know if this is any better, just to add other options.
```suggestion
Once the password for the Thing is populated in the Code tab in the UI, you must copy that into the [config files](#irobotthings-example) in order for it to persist.
```

> Without this the Roomba will appear to work temporarily, but as soon as the Things file is edited, the password is lost and the button needs to be pushed again.

Reviewer: ```suggestion
Without this, the Roomba will appear to work temporarily.
However, as soon as the Things file is edited, the password will be lost, and the button will need to be pressed again.
```
